dnfdaemon-client/commands/remove/remove.cpp

```diff
@@ -56,6 +56,10 @@ void CmdRemove::set_argument_parser(Context & ctx) {
         patterns_options);
     keys->set_short_description("List of packages to remove");
     remove->register_positional_arg(keys);
+
+    // run remove command allways with allow_erasing on
+    ctx.allow_erasing->set(libdnf::Option::Priority::RUNTIME, true);
```

Comment:
In theory `allow_erasing` could be nullptr.

Reply:
It can't. `allow_erasing` is OptionBool and value could be only `true` / `false`. What could be done is to not pass the `allow_erasing` option to the `resolve()` call when not present on the command line. But for the `remove` command I wanted it to be true so that also dependent packages could be removed.